lib/src/analyzers/lint_analyzer/rules/rules_list/prefer_correct_edge_insets_constructor/visitor.dart

```diff
@@ -0,0 +1,220 @@
+part of 'prefer_correct_edge_insets_constructor_rule.dart';
+
+const _constructorNameFromLTRB = 'fromLTRB';
+const _constructorNameSymmetric = 'symmetric';
+const _constructorNameOnly = 'only';
+
+class _Visitor extends RecursiveAstVisitor<void> {
+  final _expressions = <InstanceCreationExpression, String>{};
+
+  Map<InstanceCreationExpression, String> get expressions => _expressions;
+
+  @override
+  void visitInstanceCreationExpression(InstanceCreationExpression expression) {
+    super.visitInstanceCreationExpression(expression);
+
+    final constructorName = expression.constructorName.name?.name;
+    if (constructorName == _constructorNameFromLTRB) {
+      _validateLTRB(expression);
+    } else if (constructorName == _constructorNameSymmetric) {
+      _validateSymmetric(expression);
+    } else if (constructorName == _constructorNameOnly) {
+      _validateOnly(expression);
+    }
+  }
+
+  void _validateLTRB(InstanceCreationExpression expression) {
+    if (!expression.argumentList.arguments.every(
+      (element) => element is IntegerLiteral || element is DoubleLiteral,
+    )) {
+      return;
```

Comment:
When this condition can fulfil?

Reply:
For example  const EdgeInsets.fromLTRB(_test, _test, _test, _test)